src/Endemic/Configuration/Types.hs

```diff
@@ -299,20 +312,15 @@ data CompileConfig = CompConf
     -- Use this if you're getting crashes saying:
     -- "Exception: tests/cases/ThreeFixes.hi: openBinaryFile: resource busy (file is locked)"
     randomizeHiDir :: Bool,
-    -- | Whether or not to use bytecode or to
-    -- just interpret the resulting code.
-    -- Usuallly safe to set to true, except
-    -- when Core-to-Core plugins are involved.
+    -- | Whether or not to use bytecode or to just interpret the resulting code.
+    -- Usuallly safe to set to true, except when Core-to-Core plugins are involved.
     useInterpreted :: Bool,
-    -- | Set the timeout in microseconds for each
-    -- heck, after which we assume the check is
-    -- in an infinte loop.
+    -- | Set the timeout in microseconds for each heck, after which we assume the
+    -- check is in an infinte loop.
     timeout :: Integer,
```

Comment:
I think this should be called something like "Compiler-Timeout" 
I know this is in the Compiler-Config, but to me it's a bit too generic.

Reply:
It's the timeout for the tests and the quickcheck properties, not the compilation itself.